engine/lib/output.php

```diff
@@ -153,6 +153,12 @@ function elgg_format_attributes(array $attrs = array()) {
 
 		unset($attrs['js']);
 	}
+	
+	if (isset($attrs['data-colorbox-opts'])) {
+		if (!@json_decode($attrs['data-colorbox-opts'])) {
```

Comment:
Doesn't the user already get some sort of error indication in console? I don't like the idea of JSON validating input to views, and especially not in this function.